app.go

```diff
@@ -114,14 +113,23 @@ func main() {
 	_ = json.Unmarshal([]byte(byteValue), &pd.Rulebar)
 
 	log.Println("Server Startup at: localhost" + pd.Port)
-	nosql.MongoInit()
-	defer nosql.MongoKill()
+
+	// Attempt to connect to MongoDB with a 30 second timeout
+	// err = nosql.MongoInit(time.Second * 30)
+	// if err != nil {
+	// 	log.Printf("Could not connect the Mongo client: err = %s", err)
+	// 	os.Exit(1)
+	// }
+
+	//defer nosql.MongoKill()
 
 	http.HandleFunc("/", rootHandler)
 	http.HandleFunc("/ssrf/", makeHandler(ssrf.Handler, "ssrf"))
 	http.HandleFunc("/unvalidatedRedirect/", makeHandler(unvalidated.Handler, "unvalidatedRedirect"))
 	http.HandleFunc("/cmdInjection/", makeHandler(cmdi.Handler, "cmdInjection"))
-	http.HandleFunc("/nosqlInjection/", makeHandler(nosql.Handler, "nosqlInjection"))
+
+	// http.HandleFunc("/nosqlInjection/", makeHandler(nosql.Handler, "nosqlInjection"))
```

Comment:
will removing this handler entirely leave a dangling link in the UI?

Reply:
No, it will display the "under construction" page. This was the recommendation from @Ms-Alex